db_express_api/src/routes/maintenance.js

```diff
@@ -46,15 +46,17 @@ router.get('/maintenance/:id', function (req, res) {
 
 // Plan maintenance
 router.post('/', function (req, res) {
-    var params = req.body;
```

Comment:
Make it so you don't need to encase the things in params. so the request shouldn't look like this: body:{params:{start_date_time:...}}  but instead body:{start_date_time:...}